lib/services/downloads_service.dart

```diff
@@ -505,15 +505,24 @@ class DownloadsService {
     restartDownloads();
   }
 
+   String _convertItemTypeToTranslation(String type) {
+    switch (type) {
+      case "MusicAlbum": return "album";
+      case "Playlist": return "playlist";
+      default: return type;
+    }
+   }
+
```

Comment:
I think there might already be something similar in either `finamp_models` or `jellyfin_models`. Probably in some `toString()` method

Reply:
jup, but I changed my code to not need this anymore at all (will push later)